framework/src/Volo.Abp.Localization/Volo/Abp/Localization/LocalizableStringSerializer.cs

```diff
@@ -35,7 +35,12 @@ public LocalizableStringSerializer(IOptions<AbpLocalizationOptions> localization
 
     public virtual ILocalizableString Deserialize(string value)
     {
-        if (value.IsNullOrEmpty() ||
+        if (value == null)
```

Comment:
There are cases where the input is actually null, e.g. when defining a Feature via `FeatureDefinationProvider`, one can provide a name, but not a description

for example, see : 
https://github.com/abpframework/abp/blob/c0893cf0feef1d54e60b23ddec8e26bc92449613/framework/src/Volo.Abp.Features/Volo/Abp/Features/FeatureGroupDefinition.cs#L50-L72

Reply:
Thanks. I will check this case.

Reply:
https://github.com/abpframework/abp/pull/21206/commits/40e3e315bcd3981b1a6df5b519b9510bf7f6ece3